optaplanner-quarkus-integration/optaplanner-quarkus/runtime/pom.xml

```diff
@@ -27,16 +27,19 @@
       <artifactId>optaplanner-core</artifactId>
       <exclusions>
         <exclusion>
+          <!-- Dynamic is not compatible with GraalVM native compilation -->
           <groupId>org.kie.kogito</groupId>
           <artifactId>drools-core-dynamic</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
     <dependency>
+      <!-- Static replaces dynamic -->
       <groupId>org.kie.kogito</groupId>
       <artifactId>drools-core-static</artifactId>
     </dependency>
     <dependency>
+      <!-- TODO remove kogito-ruleunits with fix https://github.com/kiegroup/kogito-runtimes/pull/871 -->
```

Comment:
The PR has been already merged.

Reply:
Ah nice, let me try that out.

Reply:
https://github.com/kiegroup/optaplanner/pull/1017